### ISSUE TITLE
No Flip for Safari

### ISSUE THREAD

**Issue by venegu** [codenameyau/flip.js#6]
Currently, the cards do not flip in Safari. :sob:

**Comment by codenameyau** [comment]
Yup, please see #1 if you'd like to work on adding mobile support.

**Comment by venegu** [comment]
Thanks! I'll look into it because thats also pretty important. I probably should have clarified in the issue, its also not flipping on desktop Safari. 

My first thought was maybe it could be a webkit discrepancy,  but lol before I make assumptions I'll read through your source :smile:.

**Comment by codenameyau** [comment]
Give it a shot!

Be sure to read the contributing guidelines.
https://github.com/codenameyau/flip.js#contributing

**Comment by venegu** [comment]
Will do... just give me few days :mask: !
